ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/TableOptimizeItem.java

```diff
@@ -661,9 +662,9 @@ public void addNewOptimizeTasks(List<BasicOptimizeTask> newOptimizeTasks)
     }
   }
 
-  private void optimizeTasksClear(BasicOptimizeCommit optimizeCommit) {
+  public void optimizeTasksClear(Boolean refreshOptimizeStatus) {
```

Comment:
Now, `optimizeTasksClear` also clears tasks in `Executing` status, we have to handle the `optimize_task_history` of these tasks, or it will affect the quota occupation of this table.

BTW, `boolean` may be better than`Boolean`.

Reply:
OK